specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```diff
@@ -0,0 +1,1433 @@
+        "operationId": "Search_GetReverseGeocodingBatch",
+        "x-ms-examples": {
+          "A Reverse Geocoding Batch API call containing 2 Reverse Geocoding queries": {
+            "$ref": "./examples/ReverseGeocodingBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "reverseGeocodingBatchRequestBody",
+            "in": "body",
+            "description": "The list of reverse geocoding queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReverseGeocodingBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "207": {
+            "description": "Multi-Status. One or more batch items could not be processed and return different status code.",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/autosuggest": {
+      "get": {
+          "description": "Given a request with a user’s location and a partial query (called a query prefix), the Azure Maps Autosuggest API returns a list of suggested entities which the user is most likely searching for. These entities are ranked from most to least likely. Requests can be called using user locations.",
```

Comment:
The description needs to have the same format as other APIs in this file. For example, The [Get Polygon API](https://learn.microsoft.com/en-us/rest/api/maps/search-v2/get-polygon?tabs=HTTP) contains the API name (bold), pricing tier information, and the preview term with specific formats and link (see [its description definition](https://github.com/Azure/azure-rest-api-specs/blob/b8f04b910227c73d52c1042552690509f7c820ce/specification/maps/data-plane/Search/preview/2022-12-01-preview/search.json#L482)). You can compare [the previewed document of this PR](https://review.learn.microsoft.com/en-us/rest/api/documentation-preview/search/autosuggest?view=azure-rest-preview&branch=result_306513c9-46a0-47ba-acf7-919fe9fa68d3&tabs=HTTP) to the get-polygon API and modify the description according.